dao/src/test/kotlin/migrations/V73__deduplicatePackagesAndProjectsTest.kt

```diff
@@ -70,8 +70,12 @@ import org.jetbrains.exposed.sql.insert
 class V73__deduplicatePackagesAndProjectsTest : StringSpec() {
     val extension = extension(DatabaseMigrationTestExtension("72", "73"))
 
+    // TODO: The tests in this class are disabled because they are incompatible with a later migration. To enable them
```

Comment:
We could even consider to fully delete this test class. It is highly unlikely that it will be reworked. It has fulfilled its purpose and is no longer needed - keeping it will only cause maintenance effort.

Maybe this could be a general pattern for dealing with tests for migrations. Forcing them to not use any DAO classes rather complicates creating such tests.